wallet/build.gradle

```diff
@@ -50,8 +50,8 @@ dependencies {
     implementation "androidx.biometric:biometric:1.1.0"
 
     implementation "org.dashj:dashj-core:$dashjVersion"
-    implementation 'org.dashj.platform:dashpay:0.24-SNAPSHOT'
-    implementation 'org.dashj.platform:platform-core:0.24-SNAPSHOT'
+    implementation 'org.dashj.platform:dashpay:0.24-MOCK-SNAPSHOT'
+    implementation 'org.dashj.platform:platform-core:0.24-MOCK-SNAPSHOT'
```

Comment:
All data mocking is handled here.  Any data created in DashPay won't be saved there.  I didn't want to add more code to this PR requiring changes to DB tables, etc.

For now these changes will allow a username to be created or restored.